### PR TITLE
Update SpringBoot@2.6.14

### PR DESCRIPTION
**Pull request by mcornaton**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+    id 'org.springframework.boot' version '2.6.14'
     id 'io.spring.dependency-management' version '1.1.0'
     id 'jacoco'
     id 'org.sonarqube' version '3.3'
     id 'maven-publish'
 }
 
 ext {
-    springCloudVersion = '2021.0.0'
+    springCloudVersion = '2021.0.5'
```
Comment (jbern0rd) [inline build.gradle:12]: Do we still use this dependency ?

Comment (jbern0rd) [reply build.gradle:12]: I'm talking about `spring-cloud-dependencies` and `spring-cloud-starter-openfeign`

Comment (mcornaton) [reply build.gradle]: Good catch. Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/149/commits/734e525f85cf3502ff1bb4c991a3bf58d3083ef9.

**Review by jeremyjams — APPROVED** [review]

**Review by jbern0rd — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (jbern0rd) [reply build.gradle]: Thanks

**Pull request merged**